src/components/contact/contactMessage/ContactMessage.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { FC } from "react";
+import {
+  IdentityContainer,
+  InputContainer,
+  MessageContainer,
+  PanelBox,
+  SendMessageButton,
+  TitleText,
+} from "./ContactMessage.css";
+
+const ContactMessage: FC = () => {
+  return (
+    <PanelBox id="panel-box-parent">
+      <TitleText>MESSAGE</TitleText>
+      <InputContainer id="input-container">
+        <IdentityContainer id="identity-container">
+          <span>
+            <b>Name</b>
+          </span>
+          <span>
+            <b>Email</b>
+          </span>
+          <input type="text" style={{ border: "1px solid #B5B5B5" }}></input>
```

Comment:
Is there a reason why these styles are inline?

Reply:
nope! I'll put it into my "modularize" refactor 😄